src/main/webapp/app/exercises/text/assess/analytics/text-assesment-analytics.service.ts

```diff
@@ -66,6 +67,13 @@ export class TextAssessmentAnalytics {
         }
     }
 
+    private isExampleSubmissionRoute() {
+        if (!this.location) {
+            return false;
+        } else {
+            return this.location.path().includes('example-submission');
+        }
```

Comment:
```suggestion
        return !!this.location?.path().includes('example-submission');
```
Just a minor suggestion to make it more concise.